document-models/atlas-foundation/src/reducers/context.ts

```diff
@@ -6,9 +6,9 @@ export const reducer: AtlasFoundationContextOperations = {
   },
 
   removeContextDataOperation(state, action) {
-    state.originalContextData = state.originalContextData.filter(
-      (contextData) => contextData !== action.input.id,
-    );
+    // Mark the value as deleted to filter it out in the form
+    const index = state.originalContextData.indexOf(action.input.id);
+    state.originalContextData[index] = "";
```

Comment:
Here we're not deleting the URL, we're clearing it out. We must remove it or we're going to have an empty space that we could interpret as "removed", but other apps that consume the api couldn't